app/views/alchemy/admin/resources/index.html.erb

```diff
@@ -28,8 +28,19 @@
 <% end %>
 
 <div id="archive_all" class="resources-table-wrapper<%= ' with_tag_filter' if resource_has_tags || resource_has_filters %>">
-  <%= render 'alchemy/admin/resources/table_header' %>
-  <%= render 'table' %>
+  <%= render Alchemy::Admin::Resource::Table.new(resources_instance_variable, ransack_query: @query) do |table| %>
```

Comment:
Goal should be to be able to write this view as

```erb
<%= render Alchemy::Admin::Resource::Table.new(resource_handler, collection: resources_instance_variable, search_query: @query) %>
```

Reply:
Done ;)